todo_app/data/mongodb.py

```diff
@@ -0,0 +1,56 @@
+import os
+from todo_app.data.todo_item import Item
+import pymongo
+from datetime import datetime
+from bson.objectid import ObjectId
+from pprint import pprint
+
+class MongoDB:
+    def __init__(self):
+        self.client = pymongo.MongoClient(f"mongodb+srv://{os.getenv('MONGODB_USERNAME')}:{os.getenv('MONGODB_PASSWORD')}@cluster0.illz4.mongodb.net/{os.getenv('MONGODB_DATABASE_NAME')}?retryWrites=true&w=majority")
+        self.db = self.client.get_default_database()
+
+    def delete_current_database(self):
+        self.client.drop_database(self.db)
+
+    def get_collection(self, name):
+        return self.db[f'{name}'].find()
+
+    def get_items(self):
+        items = []
+        for item in self.get_collection("To Do"):
+            items.append(Item.fromMongoDb(item, "To Do"))
+        for item in self.get_collection("Doing"):
+            items.append(Item.fromMongoDb(item, "Doing"))
+        for item in self.get_collection("Done"):
+            items.append(Item.fromMongoDb(item, "Done"))
+        return items
+
+    def get_item(self, id):
+        items = self.get_items()
+        return next((item for item in items if item.id == ObjectId(id)), None)
```

Comment:
Would be faster to do things like this in the database. But it's complicated a bit by the fact that you've split your items across three collections. Would probably be better to have them all in a single collection and have a status property on each. You can then also add an index on that property to help searches.